Packs/iDefense/Integrations/iDefense_v2/iDefense_v2.py

```diff
@@ -310,14 +318,186 @@ def uuid_command(client: Client, args: dict, reliability: DBotScoreReliability)
             'TypeOfUse': res.get('last_seen_as', ''),
             'LastPublished': str(last_published_format)
         }
+        analysis_info = _enrich_analysis_result_with_intelligence(analysis_info, doc_search_client)
+
     return CommandResults(indicator=indicator,
                           raw_response=res,
                           readable_output=tableToMarkdown('Results', analysis_info))
 
 
+def hash_command(client: Client, args: dict, reliability: DBotScoreReliability, doc_search_client: Client) -> CommandResults:
```

Comment:
Hi Dor , can we ignore this . We implemented this command like `uuid_command`. It will take some more time & effort and our goal is to publish within tomorrow

Reply:
And our command works fine . no issues are arising

Reply:
if not will try my best to implement `argToList`

Reply:
@satyakidroid - it is important to accept a list of hashes as an argument to this reputation command as its standardized across all other vendors' integration. You can see the standard [here](https://xsoar.pan.dev/docs/integrations/generic-commands-reputation#file-file) where `isArray:true`.

As @dorschw mentioned, you can use the [`ip_command`](https://github.com/iDefense/content/blob/f56a6741fad560789de103297965a1cee363285d/Packs/iDefense/Integrations/iDefense_v2/iDefense_v2.py#L182) as an example.

Reply:
if I remove `isArray:true` or make it to `false`

Reply:
we are planning to get this command removed all together for now. We will look at this in future